tests/integration/support/utils.go

```diff
@@ -23,12 +23,37 @@ import (
 	"strings"
 	"time"
 
+	"github.com/optimizely/go-sdk/tests/integration/models"
+
 	"github.com/google/go-cmp/cmp"
 	"github.com/optimizely/go-sdk/pkg/config"
 
 	"gopkg.in/yaml.v3"
 )
 
+func parseListeners(value string) (listeners []interface{}) {
```

Comment:
Need to check if we can avoid this logic.

Reply:
Discussed offline, we can avoid this parsing if comparison of types doesn't cause problem. 
When comparing two objects, yaml returns `[]interface{}` type while actual object returns `[]TrackListener`. My suggestion is remove `Listener` models and compare objects using `[]interface`. will try it out and update the code.